dts/bindings/power-domain/intel,adsp-power-domain.yaml

```diff
@@ -0,0 +1,27 @@
+# Copyright (c) 2022 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+
+description: Power domains for intel ADSP
+
+compatible: "intel,adsp-power-domain"
+
+include: power-domain.yaml
+
+properties:
+    lps_handle:
+      type: phandle
+      required: true
+      description: |
+        A phandle to low power sequencer.
+        PWRCTL and PWRSTS registers used to write/read pwr gating
+        states are located there
+
+    bit_position:
```

Comment:
use `-`, not `_`, to separate words in property names

Reply:
done

Reply:
no, it's there. This snippet has "outdated" label.
```
properties:
    lps:
      type: phandle
      required: true
      description: |
        A phandle to low power sequencer.
        PWRCTL and PWRSTS registers used to write/read pwr gating
        states are located there
```

Reply:
> no, it's there

No, it's not. Please reread the requested change: https://github.com/zephyrproject-rtos/zephyr/pull/45475#discussion_r924917514

Reply:
That's embarrassing, my bad. Should be ok now.